.github/workflows/on-pull-request.yml

```diff
@@ -92,10 +92,19 @@ jobs:
           platform: linux
           tag: v0.2.2
           cache: enable
-      
-      - name: Debug print df
-        run: df -h
-        shell: bash
+
+      - name: Free disk space before build
+        run: |
+          echo "Disk space before cleanup:"
+          df -h
+          sudo rm -rf /usr/local/.ghcup
+          sudo rm -rf /opt/hostedtoolcache/CodeQL
+          sudo rm -rf /usr/local/lib/android/sdk/ndk
```

Comment:
This is gonna break the android build. The setup step goes through and installs this and Java so it can build android

Reply:
Oh or not? Hahah how?